2037.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {number[]} seats
+ * @param {number[]} students
+ * @return {number}
+ */
+var minMovesToSeat = function (seats, students) {
+  const sortedStudent = students.sort((a, b) => a - b);
+  const sortedSeat = seats.sort((a, b) => a - b);
```

Comment:
```suggestion
  const sortedStudents = [...students].sort((a, b) => a - b);
  const sortedSeats = [...seats].sort((a, b) => a - b);
```
[nitpick]It would be more readable to make those constants plural.

It would be better to create deep copies of both array before manipulating them to avoid side effects.